pyspecProcScripts/load_data.py

```diff
@@ -363,7 +378,14 @@ def proc_spincore_IR(s, fl=None):
         s.chunk("t", ["ph2", "ph1", "t2"], [2, 2, -1])
     s.setaxis("ph1", r_[0, 2.0] / 4)
     s.setaxis("ph2", r_[0, 2.0] / 4)
-    s.reorder(["ph1", "ph2"]).set_units("t2", "s")
```

Comment:
why is this removed??

Reply:
I see what you're doing below, but it seems like a messier and more verbose way of doing the same thing